content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -75,11 +75,126 @@ To setup unified service tagging in a containerized environment:
 
 #### Configuration
 
+##### Automatic version tagging for containerized environments
+You can use the `version` tag to [monitor deployments][17] and to identify faulty code deployments through [Automatic Faulty Deployment Detection][18].
+
+Datadog sets the `version` tag for you in the following priority order. If you manually set `version`, Datadog does not override your `version` value.
+
+| Priority         | Version Value |
+|--------------|------------|
+| 1    |  {your version value}       |
+| 2   | {image_tag}_{first_7_digits_of_git_commit_sha}       |
+| 3         |  {image_tag} or {first_7_digits_of_git_commit_sha} if only one is available      |
+
+Requirements: 
+- Datadog Agent Version 7.52.0 or greater
+- If running containerized and `image_tag` suffices, no further configuration necessary
```

Comment:
Ah you used my literal wording 😆 

Maybe
> If your services run in a containerized environment and `image_tag` is sufficient for tracking new version deployments, no further configuration is needed

And then
> If your services are not running in a containerized environment, or if you'd also like to have ... (same as it is now)

Reply:
Updated